CartingApp/Bill.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace CartingApp
+{
+    public class Bill
+    {
+        const double discountPercentage = 30;
+        public double totalBill;
```

Comment:
Important data holding instance variables should be converted to fields with private setters fro a security point of view